extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -56,7 +56,7 @@ class MembershipsButtonEdit extends Component {
 			products: [],
 			siteSlug: '',
 			editedProductCurrency: 'USD',
-			editedProductPrice: minimumTransactionAmountForCurrency( 'USD' ),
+			editedProductPrice: parseFloat( minimumTransactionAmountForCurrency( 'USD' ) ).toFixed( 2 ),
```

Comment:
@sixhours we have an @automattic/format-currency package that we can use. @Automattic/earn do we currently only support USD in memberships? Different locales display currency differently with different precisions and grouping markers (eg , vs .)

Reply:
Ah okay localization support for number input is also implemented unevenly in browsers, the control is kind of hard to use directly. Wonder if it'd be worth using a string then validating. 🤔 Floats are a always suspect to use when working with money

Reply:
We take quite a bit of currencies and expect to take more in the future. Not all of them have 2 decimal points.
We could try to use formatCurrency here and just pass empty string as currency symbol in the third argument (options)
https://github.com/Automattic/wp-calypso/blob/master/packages/format-currency/src/index.js#L23